ansible/inventories/sandbox/hosts

```diff
@@ -59,3 +59,15 @@ jumpbox
 solr
 jenkins
 wordpress-web
+
+[pycsw-web:children]
+pycsw-web-v2
+
+[pycsw-web-v2:children]
+catalog-next
```

Comment:
we should list the hosts explicitly. This includes catalog-harvester-next1tf.internal.sandbox.datagov.us which is not a web host.

Reply:
Is this better @adborden?
https://github.com/GSA/datagov-deploy/pull/2359